### ISSUE TITLE
A11y_VisualStudioCodeClient_Terminal_ScreenReader: In accessible terminal, NVDA does not announce the command when the focus reaches onto any line of the command's output.

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#26946]
**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver".**

-- GitHub bugs
-A11ySev4; -A11yUsable; -A11yTCS; -DesktopApp; -Visual Studio Code Client; -DesktopApp; -BM-VisualStudioCodeClient-Win32-Aug2022; -NVDA; -Win32; -Win11; -A11yeDAD; -ScreenReader; -FTP;

-- Actual:

In Accessible terminal, NVDA does not announce the command when the focus reaches onto any line of the command's output.

-- Expected:

NVDA should announce the executed command when the focus moves to any line of the command's output, allowing users to understand and track the executed commands.

-- Environment Details:​

Application Name: Visual Studio Code
Version: 1.80.0-insider (user setup)
V8: 10.8.168.25-electron.0
Windows 11 22H2 OS build 22621.1848
Screen Reader: NVDA (2023.1)

--Repro Steps:

1. Launch Visual Studio Code.
2. Tab to Terminal.
3. Execute any command, that produces correct output.
4. Turn on NVDA.
5. Then use Shift+Tab shortcut key to switch to "Accessible Terminal".
6. Move NVDA focus onto any line of the commands output.
7. Check if NVDA announcing anything about the command when the focus is on its output.

-- User Impact:

This issue affects the user experience of the terminal for NVDA users, especially those relying on auditory feedback.

"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."
